### PR TITLE
Add Camera Capture Basics Tutorial

### PR DESCRIPTION
**Pull request by CommanderFoo**
Part 1 of the Camera Capture series. Refactored the whole tutorial into parts. Changed some wording to make it a bit more beginner friendly in the last 3 sections, moved things around so the stuff you need to know to get into camera capturing is above the 3 optional sections that explain and show of how to use certain functions.

So we decided to split the tutorial into parts. Icon Generator will be part 2, and Photo Booth will be part 3. Part 2 needs to be refactored to be a standalone tutorial.

### CODE REVIEW

File: src/tutorials/camera_capture_basics.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    ```
+
+### Test the Game
+
+Test the game and see if the UI image displays the camera capture.
+
+![!First Capture](../img/CameraCapture/Basics/basics_first_capture.png){: .center loading="lazy" }
+
+### Challenge
+
```
Comment (DoubleA-MG) [inline src/tutorials/camera_capture_basics.md:125]: Have you used challenges in other tutorials? Not saying it's wrong just interested to see if it has been done before.

**Review by DoubleA-MG — COMMENTED** [review]
The first half seems beginner friendly, but the second half does seem a bit out of place knowledge-level wise. It might be better to move the memory management and capture release sections to the future lessons.

File: src/tutorials/camera_capture_basics.md (hunk 1 above)
Comment (CommanderFoo) [reply src/tutorials/camera_capture_basics.md:125]: Sort of. More of a list in the summary, not as it's own section. 

I am considering removing it though, I think I want to expand on the previous section a little.

**Comment by CommanderFoo** [comment]
> The first half seems beginner friendly, but the second half does seem a bit out of place knowledge-level wise. It might be better to move the memory management and capture release sections to the future lessons.

Yeah. The problem is that it's handy to know about releasing and refreshing. The refreshing is more important as it opens up more possibilities. We could just take these and put them into the UI ref and link them. It depends, hmm.

Will talk more about this in standup and see what would be best.

**Review by DoubleA-MG — APPROVED** [review]

**Pull request merged**